mantis-control-plane/mantis-control-plane-server/src/main/java/io/mantisrx/server/master/config/MasterConfiguration.java

```diff
@@ -51,6 +51,10 @@ public interface MasterConfiguration extends CoreConfiguration {
     @Config("mantis.master.resourceClusterProvider")
     String getResourceClusterProvider();
 
+    @Config("mantis.master.resourceClusterDefaultId")
+    @Default("mantisagent")
+    String getDefaultResourceClusterId();
+
```

Comment:
hmm, as mentioned elsewhere, why not make resourceClusterId a first class citizen of the JobDefinition / JobClusterDefinition?